### DISCUSSION TITLE
how do I use this software with my  standard WebCam  for head movement as a way to move my  mouse?

### DISCUSSION THREAD

**Discussion by kingy000** [opentrack/opentrack#1841] [Q&A]
how do I use this software with my  standard Webcam  for head movement as a way to move my  mouse? I would like to uses for outside of game like basic browsing and video editing.  also,  I would like to use it for runescape. 
I have a disability which limit my hand  movement..

**Comment by Pieloth** [comment]
Choose Input: neuralnet tracker
Choose Output:  mouse emulation, X axis set to Yaw, Y axis set to pitch
Choose Filter: NaturalMovement

Then you'll need to adjust some parameters for these settings, as well as for the options and curves mapping, to have the mouse pointer move as you want: speed, behavior, centering if needed, ...

**Reply by kingy000** [reply]
Thanks for the respond. I got the first three  you mentioned.   where  And how can I adjust the parameter  setting?  and how do I make it stop shaking?
